### PR TITLE
Vince/model

### PR DESCRIPTION
**Pull request by vliu15**
Added default dot product attention with dropout

### CODE REVIEW

File: transformer/SubLayers.py [hunk 1]
```diff
@@ -79,3 +80,34 @@ def forward(self, x):
...
+    ''' Multiplicative Attention layer '''
+
+    def __init__(self, d_model, d_hidden):
+        super().__init__()
+        self.attn_weight = nn.Linear(d_hidden, d_model)
+        self.softmax = nn.Softmax(dim=1)
+
+    def forward(self, enc_output, ses_hidden):
+        attn_vec = self.attn_weight(ses_hidden).unsqueeze(-1)
+        attn_distr = torch.bmm(enc_output, attn_vec).repeat(1, 1, enc_output.size(-1))
```
Comment (natel9178) [inline transformer/SubLayers.py:94]: Why is .repeat(1, 1, enc_output.size(-1)) necessary?

**Review by natel9178 — APPROVED** [review]

File: transformer/SubLayers.py (hunk 1 above)
Comment (natel9178) [reply transformer/SubLayers.py:94]: Repeated for batched matmul.
